packages/experiments/src/components/PersonaCoin/PersonaCoinPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { ExampleCard, IComponentDemoPageProps, ComponentPage, PropertiesTableSet } from '@uifabric/example-app-base';
+
+import { PersonaCoinExample } from './examples/PersonaCoin.Example';
+import { PersonaCoinSizeAndColorExample } from './examples/PersonaCoinSizeAndColor.Example';
+
+const PersonaCoinExampleCode = require('!raw-loader!@uifabric/experiments/src/components/PersonaCoin/examples/PersonaCoin.Example.tsx') as string;
+const PersonaCoinSizeAndColorExampleCode = require('!raw-loader!@uifabric/experiments/src/components/PersonaCoin/examples/PersonaCoinSizeAndColor.Example.tsx') as string;
+
+export class PersonaCoinPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render(): JSX.Element {
+    return (
+      <ComponentPage
+        title=" PersonaCoin"
+        componentName=" PersonaCoin"
+        exampleCards={
+          <div>
+            <ExampleCard title="PersonaCoin basic usage" code={PersonaCoinExampleCode}>
+              <PersonaCoinExample />
+            </ExampleCard>
+
+            <ExampleCard title="PersonaCoin sizes and colors" code={PersonaCoinSizeAndColorExampleCode}>
+              <PersonaCoinSizeAndColorExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={[require<string>('!raw-loader!@uifabric/experiments/src/components/PersonaCoin/PersonaCoin.types.ts')]}
+          />
+        }
+        // overview={<div />}
+        // bestPractices={<div />}
```

Comment:
cleanup

Reply:
Added some overview text and followed the rest of the experiments when it comes to `bestPractices`.